src/corelibs/U2Gui/src/util/GObjectComboBoxController.cpp

```diff
@@ -79,7 +79,10 @@ void GObjectComboBoxController::addDocumentObjects(Document* d) {
     // checks whether you need to add a new annotations table
     QString docUrl = settings.relationFilter.ref.docUrl;
     if (d->getURLString() == docUrl) {
-        connect(d->getObjectById(settings.relationFilter.ref.entityRef.entityId), SIGNAL(si_lockedStateChanged()), SLOT(sl_lockedStateChanged()));
+        GObject* seqObj = d->getObjectById(settings.relationFilter.ref.entityRef.entityId);
+        CHECK(seqObj != nullptr, );
+
+        connect(seqObj, SIGNAL(si_lockedStateChanged()), SLOT(sl_lockedStateChanged()));
```

Comment:
All affected old-style `connect`s should be replaced with function pointer variants.